src/app/Api/Controllers/AccountController.cs

```diff
@@ -85,14 +83,12 @@ public async Task<IActionResult> VerifyEmailAsync(string token)
                 return BadRequest("Token", "Token is required.");
```

Comment:
Do you mean to remove it? I remember Nikita telling us that there is an opportunity to cancel the token. In this case UserId will be null

Reply:
@osulima in case of parameter injection  into the route:
/api/{id}

id cannot be null, just an empty string. so this condition is always false. we can use .HasValue extension for example

Reply:
Done. I was looking at the wrong line...